recipe/build.sh

```diff
@@ -3,13 +3,22 @@ set -ex
 
 mkdir builddir
 
-# need to run meson first for cross-compilation case
-$PYTHON $(which meson) setup ${MESON_ARGS} \
-    -Dblas=blas \
-    -Dlapack=lapack \
-    -Duse-g77-abi=true \
-    builddir || (cat builddir/meson-logs/meson-log.txt && exit 1)
+# HACK: extend $CONDA_PREFIX/meson_cross_file that's created in
+# https://github.com/conda-forge/ctng-compiler-activation-feedstock/blob/main/recipe/activate-gcc.sh
+# https://github.com/conda-forge/clang-compiler-activation-feedstock/blob/main/recipe/activate-clang.sh
+# to use host python; requires that [binaries] section is last in meson_cross_file
+echo "python = '${PREFIX}/bin/python'" >> ${CONDA_PREFIX}/meson_cross_file.txt
+
+# meson-python already sets up a -Dbuildtype=release argument to meson, so
+# we need to strip --buildtype out of MESON_ARGS or fail due to redundancy
+MESON_ARGS_REDUCED="$(echo $MESON_ARGS | sed 's/--buildtype release //g')"
```

Comment:
@h-vetinari do you think we could change `MESON_ARGS` to use `-Dbuildtype` instead? It is always documented like that (see https://mesonbuild.com/Builtin-options.html#details-for-buildtype) and is used a lot more than `--buildtype` if you look at the Meson issue tracker (in fact, it's hard to find `--buildtype` usage at all).

Reply:
But admittedly, that's the one coming from meson-python. We could change the spelling on the activation feedstock(s), that'd be fine by me.

Reply:
Yes, that spelling change is what I meant. That will avoid having to use this hack in all feedstocks that use meson-python.